language-design.md

```diff
@@ -17,6 +17,22 @@ For each quote, describe what it was about the quote that led you pick it.
+
+I liked this quote for a couple reasons. First, it seemed like an encapsulation of a lot of what Steele was trying to say in his talk; in designing a language, one shouldn't think necessarily about exactly what the users of the language need. Rather, according to Steele, one should think about how to allow the users to make the things they need and expand the language themselves. I think this is an interesting and important point, and the quote chosen describes that really well. I also liked the quote because it seems relevant to more than just programming language design. I think it applies to life in a way, and the notion that oftentimes, the process and learning gleaned from doing a task is as important (or more so) than the task itself.
+
+2. Steele describes introducing Java as it is to a new programmer audience, saying: "Programmers would have cried, “Too big! Too much hair! I can’t deal with all that!” But in real life it has worked out fine because the users have grown with the language and learned it piece by piece, and they buy in to it because they have had some say in how to change the language" [Steele, 1998, p. 14]. In practice, we found that the second part of this claim is certainly true. Historically, to learn Java as it was developed was to learn it through experience, piece by piece. Like Steele mentioned, learning this way is natural, and models the way we actually program. At the same time though, now that Java has been fully developed and flushed out, people do need to learn it in its current size. This leads to the interesting conclusion that learning a new language that is already "large" by Steele's definition must be done by learning it piece by piece, as if it was in the process of being developed. Shoving an entire language on someone at once would be an overwhelming way to learn it. 
+
+3. "If we add hundreds of new things to the Java programming language, we will have
+a huge language, but it will take a long time to get there. But if we add just a few
+things—generic types, overloaded operators, and user defined types of light weight, for
+use as numbers and small vectors and such—that are designed to let users make and add
+things for their own use, I think we can go a long way, and much faster. We need to put
+tools for language growth in the hands of the users."
+
+This quote is similar to the first quote, in that it encapsulates a lot of the main ideas that Steele raises in his talk. I like that he mentions specific tools that can be put "in the hands of the users" like lightweight user defined types, which are concrete examples of the idea of letting the language grow on its own. As he discusses in his APL example, even a very smart individual can't possibly grow a language as well as many people all creating the libraries and functionalities that they need and sharing them. I think this quote does a good job of summarizing this idea.
 
```

Comment:
I know the Java features he lists aren't central to the point he's trying to make, but I wish he (or you) had given a more detailed explanation as to why he wants these changes specifically.